src/main/java/com/example/notes/mapper/NoteMapper.java

```diff
@@ -23,4 +24,12 @@ public NoteDto mapEntityToDto(NoteEntity noteEntity) {
                 .build();
     }
 
+    public NoteEntity mapDtoToEntity(NoteDto noteDto) {
+        return NoteEntity.builder()
+                .id(UUID.randomUUID().toString())
+                .content(noteDto.getContent())
+                .createdAt(noteDto.getCreatedAt())
+                .build();
+    }
+
```

Comment:
Perhaps, we do not need this mapper at all at this stage of the application. The mapping logic can be done on the service layer for now. Once the app is large enough and has minor duplications, we can start to extract those to separate components.

Reply:
Ok